pygmt/src/sphinterpolate.py

```diff
@@ -55,6 +57,15 @@ def sphinterpolate(data, **kwargs):
         - :class:`xarray.DataArray` if ``outgrid`` is not set
         - None if ``outgrid`` is set (grid output will be stored in file set by
           ``outgrid``)
+
+    Example
+    -------
+    >>> import pygmt
+    >>> # Load a table of Mars with longitude/latitude/radius columns
+    >>> mars_shape = pygmt.datasets.load_sample_data(name="mars_shape")
+    >>> # Perform Delaunay triangulation on the table data
+    >>> # to produce a grid with a 1m spacing
```

Comment:
`spacing=1` means 1 arc-degree spacing:
```suggestion
    >>> # to produce a grid with a 1 arc-degree spacing
```

Reply:
Oh good catch, I was looking at the `mars_shape` pandas.DataFrame which had a `radius (m)` column. But you're right that this is spherical interpolation so should be arc-degrees!